src/data/amd_milan_perf_events.rs

```diff
@@ -19,17 +19,11 @@ static CYCLES: NamedTypeCtr = NamedTypeCtr {
 lazy_static! {
     pub static ref MILAN_CTRS: Vec<NamedCtr<'static>> = [
         NamedCtr {
-            name: "stall_backend_pkc1",
-            nrs: vec![STALL_BACKEND_1],
+            name: "stall_backend",
```

Comment:
Shouldn't it still have the pkc in its name? Do whatever we do for other processors on this metric.

But having a metric without showing what its units/scale are seems incorrect.

Reply:
Oops, you're right.